### ISSUE TITLE
SystemError exception when an AppConfig dotted path specified in INSTALLED_APPS

### ISSUE THREAD

**Issue by karamanolev** [vintasoftware/django-role-permissions#52]
In older Django versions, in settings.py in INSTALLED_APPS one used to specify the name of a module. In recent Django versions (I think 1.7+) you can specify a dotted path to an AppConfig instance and I even think that's the default, since it's listed as the first option to include apps: https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications or to quote directly "To configure an application, subclass AppConfig and put the dotted path to that subclass in INSTALLED_APPS.". Specifying an app in this way breaks django-role-permissions. Here's a stack trace:

```
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/django/core/management/__init__.py", line 327, in execute
    django.setup()
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/rolepermissions/apps.py", line 12, in ready
    load_roles_and_permissions()
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/site-packages/rolepermissions/loader.py", line 14, in load_roles_and_permissions
    import_module('.permissions', app_name)
  File "/home/vagrant/virtualenvs/venv/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2249, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2199, in _sanity_check
SystemError: Parent module 'lists.apps.ListsConfig' not loaded, cannot perform relative import
```

It works when INSTALLED_APPS contains `lists`, but not when it contains `lists.apps.ListsConfig`. This breaks only django-role-permissions - if I remove django-role-permissions from INSTALLED_APPS, the rest works perfectly fine, so it's not a project error.

The workaround I've found for now is to make this AppConfig the default one by adding `default_app_config = 'lists.apps.ListsConfig'` to `lists/__init__.py` and the listing only the module name (`lists`) in INSTALLED_APPS, but ideally this should be fixed in django-role-permissions.

**Comment by filipeximenes** [comment]
Thanks for reporting @karamanolev, will investigate how to solve this. If you have some clue, please let me know and feel free to make a PR with the fix.

**Comment by karamanolev** [comment]
@filipeximenes I understand the exception, but I have no clue how to fix it. If you have some time to investigate, I would appreciate that. If not, please let me know here and I could spend an hour or two to look into it today or tomorrow.

**Comment by filipeximenes** [comment]
Thanks @karamanolev, probably wont be able to investigate this today, maybe later this week.

**Comment by filipeximenes** [comment]
Also, happy to upload it to PyPI as soon as we have a working solution

**Comment by andersonresende** [comment]
@karamanolev fixed and deployed as version 1.2.1
